tests/features/steps/payment_steps.py

```diff
@@ -7,46 +7,61 @@
 # G I V E N #
 #############
 
-@given('the following payments')
-def step_impl(context):
-	for row in context.table:
-		details = {
-			'user_name' : row['user_name'],
-			'user_email' : row['user_email'],
-			'is_linked' : row['is_linked'],
-			'card_number' : row['card_number'],
-			'card_type' : row['card_type'],
-			'expires' : row['expires'],
-		}
-		public_data = {
-			'nickname' : row['nickname'],
-			'user_id' : row['user_id'],
-			'payment_type' : row['payment_type'],
-			'details' : details
-		}
-		private_data = {
-			'is_default' : row['is_default'],
-			'is_removed' : row['is_removed'],
-			'charge_history' : row['charge_history'],
-		}
-		PaymentService.load_sample(public_data, private_data)
+@given('the following "{url}"')
+def step_impl(context, url):
+    ''' need this blank post otherwise first live post goes to dev instead of test db '''
+    context.app.post(url, data=None, content_type='application/json')
```

Comment:
it is 100% a hack. this was killing me for a while so stumbling upon this was enough to let me move on to something else. but i'm eager to hear anyone else's fix to the problem